src/applications/vaos/components/layout/InPersonLayout.jsx

```diff
@@ -98,7 +121,7 @@ export default function InPersonLayout({ data: appointment }) {
         <span>Location: {clinicPhysicalLocation || 'Not available'}</span>{' '}
         <br />
         {facilityPhone && (
-          <FacilityPhone heading="Clinic phone:" contact={facilityPhone} />
+          <FacilityPhone heading="Phone:" contact={facilityPhone} />
```

Comment:
Hmm, the spec seems to prefer clinic phone and falls back to facility phone if that's missing. Do we need to update the logic here?

<img width="344" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/2030323/595d1c5b-5cb5-46ce-957a-e2ecad2ef346">

Reply:
@JunTaoLuo All the phone number issues are in another issue that @jenniemc is working on (https://github.com/department-of-veterans-affairs/va.gov-team/issues/80375)

Reply:
Ah I see it's tracked in the same issue. Thanks!